dist/index.js

```diff
@@ -2875,7 +2875,7 @@ module.exports = function (obj) {
+module.exports = {"_args":[["got@9.6.0","/Users/20012243/Dev/vault-action"]],"_from":"got@9.6.0","_id":"got@9.6.0","_inBundle":false,"_integrity":"sha512-R7eWptXuGYxwijs0eV+v3o6+XH1IqVK8dJOEecQfTmkncw9AV4dcw/Dhxi8MdlqPthxxpZyizMzyg8RTmEsG+Q==","_location":"/got","_phantomChildren":{},"_requested":{"type":"version","registry":true,"raw":"got@9.6.0","name":"got","escapedName":"got","rawSpec":"9.6.0","saveSpec":null,"fetchSpec":"9.6.0"},"_requiredBy":["/"],"_resolved":"https://registry.npmjs.org/got/-/got-9.6.0.tgz","_spec":"9.6.0","_where":"/Users/20012243/Dev/vault-action","ava":{"concurrency":4},"browser":{"decompress-response":false,"electron":false},"bugs":{"url":"https://github.com/sindresorhus/got/issues"},"dependencies":{"@sindresorhus/is":"^0.14.0","@szmarczak/http-timer":"^1.1.2","cacheable-request":"^6.0.0","decompress-response":"^3.3.0","duplexer3":"^0.1.4","get-stream":"^4.1.0","lowercase-keys":"^1.0.1","mimic-response":"^1.0.1","p-cancelable":"^1.0.0","to-readable-stream":"^1.0.0","url-parse-lax":"^3.0.0"},"description":"Simplified HTTP requests","devDependencies":{"ava":"^1.1.0","coveralls":"^3.0.0","delay":"^4.1.0","form-data":"^2.3.3","get-port":"^4.0.0","np":"^3.1.0","nyc":"^13.1.0","p-event":"^2.1.0","pem":"^1.13.2","proxyquire":"^2.0.1","sinon":"^7.2.2","slow-stream":"0.0.4","tempfile":"^2.0.0","tempy":"^0.2.1","tough-cookie":"^3.0.0","xo":"^0.24.0"},"engines":{"node":">=8.6"},"files":["source"],"homepage":"https://github.com/sindresorhus/got#readme","keywords":["http","https","get","got","url","uri","request","util","utility","simple","curl","wget","fetch","net","network","electron"],"license":"MIT","main":"source","name":"got","repository":{"type":"git","url":"git+https://github.com/sindresorhus/got.git"},"scripts":{"release":"np","test":"xo && nyc ava"},"version":"9.6.0"};
```

Comment:
`/Users/20012243/Dev/vault-action` is this intended?

Reply:
Hello @devon-kim thx for reviewing, 

tl;dr : absolutely not.

I am very far from an expert in nodejs stuff, and I didn't know what dist/index.js was, and I just kept @sfauvart commit as is. 

A colleague just explain me it is a compiled version of action.js with `npm run build`.

If I run `build`, I have a similar result with a `...[["got@9.6.0","/Users/antoinemeausoone/Projects/...`, so I don't know how to have a clean compiled index.js ?

Reply:
Dist getting regenerated is normal, user paths getting included is not. I'm using [Zeit's ncc tool](https://github.com/zeit/ncc) if you want to check the issues and see if someone else has seen this.

Reply:
Did you guys figure this out? I was working on a feature and noticed I was getting the same behavior (local path getting injected for some reason) when building. So far couldn't find any additional info in ncc's issues, but I'll keep looking. Thanks!

Reply:
OK I updated `got` to `^10.2.2` ,and compiled index.js. Hope it's ok. Thx @gVirtu

Reply:
There's a tiny caveat though, some tests might need adapting to the new got parameter semantics, instead of:
```
body: <some body>
json: true
```

It now expects something of the form:
```
json: <some body>
```